base_layer/wallet_ffi/README.md

```diff
@@ -35,11 +35,23 @@ For macOS Mojave additional headers need to be installed, run
 ```Shell Script
 open /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg
 ```
-and follow the prompts
+and follow the prompts.
+
+For Catalina, if you get compilation errors such as these:
+
+    xcrun: error: SDK "iphoneos" cannot be located 
+    xcrun: error: unable to lookup item 'Path' in SDK 'iphoneos'
+
+Switch the XCode app defaults with:
+
+    sudo xcode-select --switch /Applications/Xcode.app
 
 ## Android Dependencies
 
-Download the [Android NDK Bundle](https://developer.android.com/ndk/downloads)
+Install [Android Studio](https://developer.android.com/studio) and then use the SDK Manager to install the Android NDK
+along with the SDK of your choice (Android Q is recommended). The command line tools are also very handy.
```

Comment:
A side note, the command line tools don't work out of the box in a normal terminal (at least when I installed them), there were extra steps to get this working. Can't remember exactly what I did though. Using them inside Android Studio should be fine.

Reply:
Packages that can be checked to be installed is (under SDK tools tab):
* NDK (Side by side)
* Android SDK Command-line Tools (latest)
* Android SDK Platform Tools
* Android SDK Tools
* CMake